core/domain/user_query_services_test.py

```diff
@@ -62,9 +67,10 @@ def setUp(self):
         self.user_query_model_2.update_timestamps()
         self.user_query_model_2.put()
 
-    def test_get_user_query_returns_user_query(self):
+    def test_get_user_query_returns_user_query(self) -> None:
         user_query = user_query_services.get_user_query(self.USER_QUERY_1_ID)
 
+        assert user_query is not None
```

Comment:
add a comment, ditto elsewhere in this file.

Reply:
Added.